src/frontend/js/widgets/Dashboard/components/FilterOrganization/index.tsx

```diff
@@ -3,32 +3,53 @@ import { Select, SelectProps } from '@openfun/cunningham-react';
 import { useEffect } from 'react';
 import { useOrganizations } from 'hooks/useOrganizations';
 import { Spinner } from 'components/Spinner';
+import { OPTION_VALUE_ALL } from '../FiltersBar';
 
 export const messages = defineMessages({
   organizationFilterLabel: {
     defaultMessage: 'Organization',
     description: 'Use as organization filter label',
     id: 'components.ListFilterOrganization.organizationFilterLabel',
   },
+  allOrganizationOption: {
+    defaultMessage: 'All organizations',
+    description: 'Use as organization filter option label for "all organizations"',
+    id: 'components.ListFilterOrganization.allOrganizationOption',
+  },
 });
 
 interface FilterOrganizationProps {
   defaultValue?: string;
   onChange: ({ organization_id }: { organization_id?: string }) => void;
+  canSelectAll?: boolean;
 }
 
-const FilterOrganization = ({ defaultValue, onChange }: FilterOrganizationProps) => {
+const FilterOrganization = ({
```

Comment:
I wonder if the `clearable` option of the Cunningham Select is not enough to allow user to remove organization filter.